### ISSUE TITLE
implement shopt -s cdable_vars

### ISSUE THREAD

**Issue by andychu** [oilshell/oil#578]
From #574 

This has a weird syntax, but it should be a simple to implement, and if someone uses it, sure why not

**Comment by Iam-samyog** [comment]
Hey! Can I work ont this?

**Comment by andychu** [comment]
Yes sure -- please start a thread on Zulip if you need help navigating the code

**Comment by andychu** [comment]
We always start with tests to "spec out" the the feature, e.g. https://github.com/oils-for-unix/oils/wiki/Spec-Tests

**Comment by Iam-samyog** [comment]
Thanks,  I will start by looking into the Spec Tests.  I'll also hop onto Zulip to introduce myself and ask a few clarifying questions about the codebase layout. Thank you!
